cms/envs/devstack.py

```diff
@@ -152,7 +152,7 @@ def should_show_debug_toolbar(request):
 ########################### OAUTH2 #################################
 JWT_AUTH.update({
     'JWT_SECRET_KEY': 'lms-secret',
-    'JWT_ISSUER': '{}/oauth2'.format(LMS_ROOT_URL),
+    'JWT_ISSUER': 'http://edx.devstack.lms:18000/oauth2',
```

Comment:
I would actually just delete this line, so that devstack uses the values from `production.py`.

Reply:
Tested this and it seems to work